### PR TITLE
Set trust_level for over 20 rep users after 30secs

### PR DESCRIPTION
**Pull request by iHiD**

### CODE REVIEW

File: app/commands/user/set_discourse_trust_level.rb [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+
+    discourse_user_id = client.by_external_id(user.id)['id']
+    client.update_trust_level(discourse_user_id, level: 2)
+  end
+
+  def client
+    DiscourseApi::Client.new("https://forum.exercism.org").tap do |client|
+      client.api_key = Exercism.secrets.discourse_api_key
+      client.api_username = "system"
+    end
```
Comment (ErikSchierboom) [inline app/commands/user/set_discourse_trust_level.rb:20]: Did you consider making this part of `config`, like the GitHub client?

File: app/commands/user/set_discourse_trust_level.rb [hunk 2]
```diff
@@ -0,0 +1,19 @@
+class User::SetDiscourseTrustLevel
+  include Mandate
+
+  initialize_with :user
+
+  def call
+    return unless user.reputation >= 20
+
+    discourse_user_id = client.by_external_id(user.id)['id']
```
Comment (ErikSchierboom) [inline app/commands/user/set_discourse_trust_level.rb:9]: Do we need to guard for the exceptional case where the client could not be found?

**Review by ErikSchierboom — APPROVED** [review]

File: app/commands/user/set_discourse_trust_level.rb (hunk 1 above)
Comment (iHiD) [reply app/commands/user/set_discourse_trust_level.rb:20]: As this is the only place we're using it, not so far. But if we start doing more with it then yeah, that'd make sense. So if you're retrieving stuff elsewhere, feel free to do that 🙂

File: app/commands/user/set_discourse_trust_level.rb (hunk 2 above)
Comment (iHiD) [reply app/commands/user/set_discourse_trust_level.rb:9]: Yeah, good shout. I'll do that.

**Pull request merged**